### PR TITLE
Create 2020-02-21-CFP-PrestaShop-Day-Paris-2020.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+tags: [contribute, events, community]
+---
+
+
+Last year, a developer space was available at the PrestaShop Day Paris. This year, this space grows and will include an area for conferences and workshops dedicated to active contributors to the open source project.
+
+The event will take place in Paris, at "Palais Brongniart", June the 4th of 2020. More informations:  
+https://events.prestashop.com/prestashop-day-paris-2020/fr/
+
+## Thèmes
```
Comment (SimonGrn) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
## Themes
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Below is a list of themes for the call of papers that would be very appreciated this year.
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
Below is a list of themes for the [call of papers](https://en.wikipedia.org/wiki/Academic_conference#Organizing_an_academic_conference) that would be very appreciated this year.
```

I suggest providing a reference for people who dont know this word

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+**Technique:**
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: I suggest "Development" or "Developers"

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+- FO or BO integration
+- Headless, PWA, API /. web-services
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: Why this `/.` ?

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+- Information systems
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md:26]: Quite vague

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+- Migration to PrestaShop
+- Performance, tests and quality
+
+**Project and community:**
+
+- Contribute to PrestaShop
+- Use of GitHub
+- Open source culture
+
+Do not hesitate to propose a topic around the PrestaShop project. All submissions will be studied.
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
Do not hesitate to submit a topic around the PrestaShop project. All submissions will be examined.
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+The target is to share and exchange knowledge, best practices and know how.
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
The objective is to share and exchange knowledge, best practices and know how.
```

Are you a soldier 😋 ?

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+Consequently, all workshops, talks and slides must be in French -- "et sans faute d'orthographe".
+
+## Agenda and program selection
+
+13/03: End of the first round  
+16/03: Announcement of the first speakers  
+10/04: Closing of the CFP and the form  
+20/04: Announce of the complete program
+
+A first selection of submissions will be done on March 13th of 2020. And a first communication should done around March the 16th of 2020 to announce the first speakers. If you want to be selectionned, don't be late.
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: `And a first communication should done` => missing a verb

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+## Formats
+
+Three conference formats are proposed in 2020:
+
+- 45min workshop
+- 30min talk
+- 5min lightning talk
+
+In order to respect the program, the speakers and the attendees at the show, no additional time will be allocated. Therefore, talks must be very well prepared.
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: `In order to respect the program`
Please 😞 this is copy-paste from French to English.
Nobody says this in English.

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+- This will be our very first CFP
+
+Consequently, all workshops, talks and slides must be in French -- "et sans faute d'orthographe".
+
+## Agenda and program selection
+
+13/03: End of the first round  
+16/03: Announcement of the first speakers  
+10/04: Closing of the CFP and the form  
+20/04: Announce of the complete program
```
Comment (matks) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
20/04: Announcement of the complete program
```
Noun, not verb

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
**Development:**
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: typo.
```suggestion
- Headless, PWA, API / web-services
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 5 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md:26]: yes. a lot of partners might buy a sponsored talk about integration in an existing information system, so I had to make this compromise.

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 7 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: the aim, ...

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 8 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
A first selection of submissions will be done on March 13th of 2020. And a first communication should be done around March the 16th of 2020 to announce the first speakers. If you want to be selected, don't be late.
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 9 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
By respect for the program, the speakers and the attendees at the show, no additional time will be allocated. Therefore, talks must be very well prepared.
```

**Comment by ttoine** [comment]
@matks please let me know what do you think about the suggestions, in answer to your comments.

thank you very much for the review. and sorry for the mistakes. this week has been horrible and tiring.

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "Call for paper for the PrestaShop Day Paris 2020"
```
Comment (PierreRambaud) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
title:  "Call for Papers curently open for the PrestaShop Day Paris 2020"
```

Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: Nice catch, I forgot the "s" to "Papers".
What do you think of:
```suggestion
title:  "Call for Papers for the PrestaShop Day Paris 2020 is open"
```

Comment (PierreRambaud) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: "for" "for", it's redundant

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, events, community]
+---
+
+
+Last year, a developer space was available at the PrestaShop Day Paris. This year, this space grows and will include an area for conferences and workshops dedicated to active contributors to the open source project.
+
+The event will take place in Paris, at "Palais Brongniart", June the 4th of 2020. More informations:  
+https://events.prestashop.com/prestashop-day-paris-2020/fr/
```
Comment (PierreRambaud) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
(https://events.prestashop.com/prestashop-day-paris-2020/fr/)[https://events.prestashop.com/prestashop-day-paris-2020/fr/]
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+## Themes
+
+Below is a list of themes for the [call of papers](https://en.wikipedia.org/wiki/Academic_conference#Organizing_an_academic_conference) that would be very appreciated this year.
+
+**Technique:**
+
+- FO or BO integration
```
Comment (PierreRambaud) [inline news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
- Front Office or Back Office integration
```

File: news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md (hunk 11 above)
Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: CFP for the PrestaShop Day Paris 2020 is open
?

Comment (PierreRambaud) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]: CFP isn't explicit for everyone.

Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
title:  "PrestaShop Day Paris 2020: Call for paper is open"
```

Comment (ttoine) [reply news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md]:
```suggestion
title:  "PrestaShop Day Paris 2020: Call for Papers is open"
```

**Pull request merged**